client/src/client/client.go

```diff
@@ -53,8 +53,42 @@ func GetUserInput(text *string, reader io.Reader) error {
 // if any conn write call fails, return how many bytes you wrote and an error
 // if everything works out fine, return how many bytes you wrote and a nil error
 func SendToProducer(buf []byte, addr string) (int, error) {
-	// TODO
-	return 0, errors.New("Incomplete function")
+	// Opens a connection, if connection fails, return 0 and error
+	conn, err:= net.Dial("tcp", addr)
+	if err != nil {
+		conn.Close()
+		return 0, err
```

Comment:
For these error returns, it would be useful for future logging debugging statements if they had custom messages describing what was wrong. I'm requestin that if errors are not `nil` that they be `errors.New("error description")`. This is applying to all functions moving forward and we'll retroactively correct this once we begin testing.